clients/src/main/resources/common/message/DescribeTopicPartitionsResponse.json

```diff
@@ -59,8 +59,8 @@
     { "name": "NextCursor", "type": "Cursor", "versions": "0+", "nullableVersions": "0+", "default": "null",
       "about": "The next topic and partition index to fetch details for.", "fields": [
       { "name": "TopicName", "type": "string", "versions": "0+",
```

Comment:
This is a bit of a mess. This line and the next both include "versions". I suggest moving the entityType onto line 61 and just having the about description on line 62.